sw/device/lib/testing/pwrmgr_testutils.c

```diff
@@ -33,3 +33,20 @@ bool pwrmgr_testutils_is_wakeup_reason(const dif_pwrmgr_t *pwrmgr,
           wakeup_reason.types == kDifPwrmgrWakeupTypeRequest) &&
          wakeup_reason.request_sources == reasons;
 }
+
+void pwrmgr_testutils_enable_usb_clock_in_active_state(
+    const dif_pwrmgr_t *pwrmgr) {
+  dif_pwrmgr_domain_config_t cfg;
+  CHECK_DIF_OK(dif_pwrmgr_get_domain_config(pwrmgr, &cfg));
+  cfg |= kDifPwrmgrDomainOptionUsbClockInActivePower;
+  CHECK_DIF_OK(dif_pwrmgr_set_domain_config(pwrmgr, cfg, kDifToggleEnabled));
+}
+
+void pwrmgr_testutils_enable_usb_clock_in_low_power_state(
```

Comment:
This is an overreach, since it both sets the usb control and causes shallow sleep. It also causes a wait for a full cdc sync: we normally make all configs writes and then have a single wait.

Reply:
Agreed and removed.